project_national/fact_sheets_category_2.yml

```diff
@@ -0,0 +1,662 @@
+            - Infiltration|5 ACH50
+            - Infiltration|4 ACH50
+        costs:
+          - value: 0.50 # To be decided
+                        # $0.50/ft2 is from LBNL data for general air sealing (https://doi.org/10.20357/B7FP4D).
+                        # Interpolated between 20% and 40% reductions to get $0.44/ft2,
+                        # then used 1.14 multiplier to inflate from Jan 2019 to Mar 2022
+            multiplier: Floor Area, Conditioned (ft^2)
+        lifetime: 60 # years effective useful life for opaque envelope components, from https://www.nyserda.ny.gov/-/media/Files/Programs/energy-code-training/Residential-Cost-Analysis-Report.pdf
+      - &infiltration_IECC2021_add_ERV
+        option: Mechanical Ventilation|ERV, 72%
+        costs:
+          - value: 0.50 # To be decided
+            multiplier: Floor Area, Conditioned (ft^2)
+        lifetime: 60
+      - &roof_insulation_to_R30
+        option: Insulation Roof|Finished, R-30
+        apply_logic: 
+          - or:
+            - Geometry Attic Type|Finished Attic or Cathedral Ceilings
+            - Geometry Building Level MF|Top
+          - or:
+            - Insulation Roof|Finished, Uninsulated
+            - Insulation Roof|Finished, R-7
+            - Insulation Roof|Finished, R-13
+            - Insulation Roof|Finished, R-19
+        costs:
+          - value: 6.02 # $/ft2 (4.3* $/ft2 * 1.4 inflation) https://remdb.nrel.gov/measures?group_id=13&component_type_id=395&actionId=2621&bcId=6283
+            multiplier: Roof Area (ft^2)
+        lifetime: 60 # years effective useful life for opaque envelope components, from https://www.nyserda.ny.gov/-/media/Files/Programs/energy-code-training/Residential-Cost-Analysis-Report.pdf
+  
+  - upgrade_name: Advanced Enclosure with mix of ERVs and exhaust-only ventilation
+    #Up
+    options:
+      #Light Enclosure Measures
+      - *attic_insulation_IECC_CZ1A
+      - *attic_insulation_IECC_CZ2A-2B-3A-3B-3C
+      - *attic_insulation_IECC_CZ4A-4B-4C-5A-5B-6A-6B-7A-7B
+      #Intermediate Enclosure Measures
+      - *ducts_category1
+      - *ducts_category2
+      - *ducts_category3
+      - *ducts_category4
+      #- *ducts_category5
+      #- *ducts_category6
+      - *ducts_category7
+      - *ducts_category8
+      - *Drill-and-fill
+      - *foundation_wall_insulation
+      - *foundation_wall_insulation_finished_basement
+      - *rim_joist_insulation
+      - *seal_vented_crawlspaces
+      #EnergyStar Windows
+      - *EnergyStar_window_northern
+      - *EnergyStar_window_north_central
+      - *EnergyStar_window_south_central
+      - *EnergyStar_window_southern
+      - *infiltration_30pct_reduction_windows
+      - *infiltration_15pct_reduction_windows
+      
+      #1” XPS (R-5/in) Exterior wall insulation for dwelling units with wall insulation of less than R-19  
+      - *exterior_wall_insulation
+      - *infiltration_IECC2021_with_ERV_added_IECC_CZ1A-2A-2B
```

Comment:
Since this option is just the insulation and not the ERV, can we rename the anchor to reflect that please?

Reply:
done